### PR TITLE
java admin ClientResource.update() returns 409

### PR DESCRIPTION
**Pull request by Caldfir**
[KEYCLOAK-8620]
If `AuthorizationService.enable(boolean)` is called multiple times, then it should not re-create the resource server when it already exists. Attempting to create when it already exists would cause a conflict with the existing resources.

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/ClientTest.java [hunk 1]
```diff
@@ -223,6 +223,42 @@ public void updateClient() {
...
+      client.setClientId("my-app-oldname");
+      client.setPublicClient(false);
+      client.setBearerOnly(false);
+      client.setStandardFlowEnabled(true);
+      client.setImplicitFlowEnabled(false);
+      client.setDirectAccessGrantsEnabled(true);
+      client.setServiceAccountsEnabled(true);
+      client.setAuthorizationServicesEnabled(true);
+      client.setRedirectUris(Collections.singletonList("/"));
+      assertEquals(201, realm.clients().create(client).getStatus());
```
Comment (mposolda) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/ClientTest.java]: It is good to close the response and also it's possible to retrieve the newly created ID with something like this:
````
Response response = realm.clients().create(client);
String clientUUID = ApiUtil.getCreatedId(response);
response.close();
````

**Review by mposolda — COMMENTED** [review]
Thanks for the PR. I am assigning to @pedroigor as he is more familiar with authorization services.

I just left one minor inline comment regarding test. Also one general comment is, that commits should be squashed, so there is 1 commit per PR.

**Comment by pedroigor** [comment]
@Caldfir, I think this issue was already fixed by https://issues.jboss.org/browse/KEYCLOAK-8625. Which version of Keycloak are you using ?

**Comment by Caldfir** [comment]
@pedroigor I was using 4.5.0 - I will try again with 4.6+ and get back to you.

**Comment by pedroigor** [comment]
Thanks for your contribution, but this issue should be fixed by https://issues.jboss.org/browse/KEYCLOAK-8625. I'm closing this PR, if you think the issue still persists, please let us know.

**Pull request closed without merging**